Sources/hostmgr-helper/VirtualMachineSlot.swift

```diff
@@ -48,11 +48,12 @@ class VirtualMachineSlot: NSObject, ObservableObject, VZVirtualMachineDelegate {
         do {
             let virtualMachine = try await launchConfiguration.setupVirtualMachine()
             virtualMachine.delegate = self
+            self.virtualMachine = (virtualMachine, launchConfiguration)
```

Comment:
I wanted to couple this state because they both should be mutually inclusive outside of the `start` call. A few type felt too heavy though.